include/swift/Option/Options.td

```diff
@@ -286,6 +286,10 @@ def disable_bridging_pch : Flag<["-"], "disable-bridging-pch">,
   Flags<[HelpHidden]>,
   HelpText<"Disable automatic generation of bridging PCH files">;
 
+def e : Separate<["-"], "e">,
+  Flags<[FrontendOption, NoBatchOption, DoesNotAffectIncrementalBuild]>,
+  HelpText<"Evaluate one line of Swift code. Multiple -e's allowed.">;
+
```

Comment:
I'm fuzzy on what the various options here mean, so I kinda guessed at making this `Separate` here. 

`Joined` seems to be reserved for cases where an `=` is used, (like in `-I=<directory>`) so that didn't seem appropriate, and `JoinedOrSeparate` seems to imply one could use `-e'print("hello")'` which doesn't seem right either.

So `Separate` seemed like the best choice.

Are the `Flags` correct? Kinda guessing here as well.

Also, I winged it on the help text verbiage here. It's paraphrased from the output of the `-e` equivalents in `ruby`, `python` & `perl`, which they seem to borrow from each other from what I can tell.

 Let me know if it needs to be changed and I'll try and come up with something different.

Reply:
This part looks fine!